.travis.yml

```diff
@@ -1,15 +1,149 @@
+      packages:
+        - python
+        - python-pip
+  install:
+    - pip install awscli --upgrade --user
+  script:
+    - npm run doc-prepare
+    - npm run doc-build
+  deploy:
+    provider: script
+    script:
+      - npm run doc-upload
+    skip_cleanup: true
+  after_deploy:
+    - npm run doc-cloudfront
+
+# ------------------------
+# Jobs configuration
+# ------------------------
+jobs:
+  include:
+    # ---------------------------------------
+    # Unit Tests & Linters
+    # ---------------------------------------
+    - name: PHP 7 (with coverage)
+      stage: Unit Tests & Linters
+      <<: *cross_tests
+      after_success:
+        - bash <(curl -s https://codecov.io/bash)
+
+    - name: Dead link check
+      stage: Unit Tests & Linters
+      language: node_js
+      node_js: 10
+      cache:
+        directories:
+          - "$HOME/.gem/specs"
+          - ".doc/framework/node_modules"
+      before_script:
+        - npm run doc-prepare
+        - npm run --prefix .doc/framework clone-repos
+      script:
+        - gem install typhoeus
+        - HYDRA_MAX_CONCURRENCY=20 npm run --prefix .doc/framework dead-links
+
+    - name: Documentation test
+      stage: Unit Tests & Linters
+      language: go
+      go: 1.12.x
+      cache:
+        directories:
+          - "$HOME/.cache/go-build"
+          - "$HOME/gopath/pkg"
+      script:
+        - docker-compose -f .ci/doc/docker-compose.yml run doc-tests node index
+
+    # -----------------------------------------------
+    # Cross-platform tests
+    # -----------------------------------------------
+    - <<: *cross_tests
+      name: PHP 5.4
+      stage: Retro
+      php: 5.4
+
+    - <<: *cross_tests
+      name: PHP 5.5
+      stage: Unit Tests & Linters
+      php: 5.5
+      
+    - <<: *cross_tests
+      name: PHP 5.6
+      stage: Unit Tests & Linters
+      php: 5.6
+      
+    - <<: *cross_tests
+      name: PHP 7.0
+      stage: Unit Tests & Linters
+      php: 7.0
+
+    # -----------------------------------------------
+    # Deployments
+    # -----------------------------------------------
+    - <<: *doc-deploy
+      name: Documentation to next-docs.kuzzle.io
+      if: branch =~ /^[0-9]+-dev$/
+      env:
+        - *doc-deploy-env
+        - S3_BUCKET=docs-next.kuzzle.io
+        - CLOUDFRONT_DISTRIBUTION_ID=E2ZCCEK9GRB49U
+
+    - <<: *doc-deploy
+      name: Deploy docs.kuzzle.io
+      if: branch =~ /^master|[0-9]+-stable$/
+      env:
+        - *doc-deploy-env
+        - S3_BUCKET=docs.kuzzle.io
+        - CLOUDFRONT_DISTRIBUTION_ID=E3D6RP0POLCJMM
+
+# ------------------------
+# Stages configuration
+# ------------------------
+stages:
+  - name: Unit Tests & Linters
+    if: type =~ /(cron|pull_request)/ OR (type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/)
+  - name: Retro
+    if: type = cron
+  - name: Deployments
+    if: type = push AND branch =~ /^master|[0-9]+-(dev|stable)$/
+
+# ---------------------------------------
```

Comment:
Remove all the Notifications section sinceTravis notification process does not match our needs